apps/odyssey-stats/src/widget/highlights.jsx

```diff
@@ -110,18 +114,47 @@ export default function Highlights( { siteId, gmtOffset, odysseyStatsBaseUrl } )
 		queryDate
 	);
 
+	const headingTitle = translate( '7 Day Highlights' );
+	const topPostsAndPagesTitle = translate( 'Top Posts & Pages' );
+	const topReferrersTitle = translate( 'Top Referrers' );
+
+	const moduleTabs = [
+		{
+			value: 'topPostsAndPages',
```

Comment:
Could we use this value directly in `useState`? I'd like for us to avoid needing to update both L96 and L123 if we ever need to update this.